hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -760,6 +760,41 @@ public String getObjectInfo() {
         '}';
   }
 
+
+  public boolean isKeyInfoSame(OmKeyInfo omKeyInfo, boolean checkPath,
```

Comment:
I think `isKeyInfoSame` is very specific to snapshot diff and should be added to snapshot util.

Reply:
It is better this way to put in key info if some field is added in omKeyInfo it should impact the equals method as well thus impacting snapshot diff.

Reply:
Fair enough.